### PR TITLE
feat(core): adds angular 19 support

### PR DESCRIPTION
**Pull request by pavankjadda**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,31 +53,31 @@
...
-    "@angular/router": "^18.0.1",
+    "@angular/animations": "^19.0.1",
+    "@angular/common": "^19.0.1",
+    "@angular/compiler": "^19.0.1",
+    "@angular/core": "^19.0.1",
+    "@angular/forms": "^19.0.1",
+    "@angular/platform-browser": "^19.0.1",
+    "@angular/platform-browser-dynamic": "^19.0.1",
+    "@angular/router": "^19.0.1",
     "@schematics/angular": "^18.0.2",
```
Comment (timdeschryver) [inline package.json:64]: This should also be updated to v19

**Review by timdeschryver — COMMENTED** [review]
Nice, thanks @pavankjadda !

File: package.json [hunk 2]
```diff
@@ -53,31 +53,31 @@
...
+    "@angular-eslint/builder": "^19.0.0",
+    "@angular-eslint/eslint-plugin": "^19.0.0",
+    "@angular-eslint/eslint-plugin-template": "^19.0.0",
+    "@angular-eslint/schematics": "^19.0.0",
+    "@angular-eslint/template-parser": "^19.0.0",
+    "@angular/cli": "^19.0.2",
+    "@angular/compiler-cli": "^19.0.1",
     "@evilmartians/lefthook": "^1.0.3",
     "@types/jasmine": "^4.0.0",
     "@types/node": "^16.11.7",
```
Comment (timdeschryver) [inline package.json:83]: To resolve the build error, the `@types/node` version also needs to be upgraded.

File: package.json (hunk 1 above)
Comment (pavankjadda) [reply package.json:64]: Will do

File: package.json (hunk 2 above)
Comment (pavankjadda) [reply package.json]: Done

**Review by timdeschryver — APPROVED** [review]

**Pull request merged**